lib/widgets/ScheduleBar.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+class ScheduleBar extends StatelessWidget implements PreferredSizeWidget {
+
+  ScheduleBar(this.textColor, this.backgroundColor);
+  final Color textColor;
+  final Color backgroundColor;
+
+  static final double prefHeight = 80;
```

Comment:
The schedule bar seems a bit too long compared to the designs. 
<img width="306" alt="Screen Shot 2021-04-06 at 1 28 32 AM" src="https://user-images.githubusercontent.com/17262023/113663091-671a8d00-9677-11eb-8f22-37e145bf3ae2.png">

Reply:
Fixed, thanks!